pkg/controllers/dynakube/oneagent/daemonset/daemonset.go

```diff
@@ -380,3 +385,22 @@ func (dsInfo *builderInfo) getReadinessProbe() *corev1.Probe {
 
 	return defaultProbe
 }
+
+// isRootFsReadonly checks if the given version of the OneAgent supports the `ReadOnlyRootFilesystem` securityContext setting.
+// if the version is not set, ie.: unknown, we  consider the OneAgent to support `ReadOnlyRootFilesystem`.
+func (dsInfo *builderInfo) isRootFsReadonly() bool {
+	if dsInfo.dynakube != nil &&
+		dsInfo.dynakube.NeedsReadOnlyOneAgents() &&
+		dsInfo.dynakube.OneAgentVersion() != "" {
+		agentSemver, err := dtversion.ToSemver(dsInfo.dynakube.OneAgentVersion())
+		if err != nil {
+			log.Error(err, "Unable to determine OneAgent version to enable readonly pod filesystem, skipping")
```

Comment:
I would prefer to use Debug instead of error and add more context to it, because we do nothing with it :)

Reply:
done